### PR TITLE
Add version and build-number outputs

### PR DESCRIPTION
**Pull request by tatsuz0u**

### CODE REVIEW

**Comment by tatsuz0u** [comment]
Not quite familiar with JavaScript...
I modified `/dist/index.js` manually by the way.

**Comment by tatsuz0u** [comment]
Use case: https://github.com/tatsuz0u/EhPanda/blob/815ff373048c8bab49eda1328f271259752ca7de/.github/workflows/deploy.yml#L96

**Comment by yanamura** [comment]
`npm run build` will make /dist/index.js

File: dist/index.js [hunk 1]
```diff
@@ -984,12 +987,20 @@ if (version) {
 }
 
 if (!buildNumber) {
+    execCommand(`agvtool what-version -terse`, {
+        listeners: { stdout: (data) => {
+            core.setOutput('build-number', +data + 1)
+        }}
+    })
```
Comment (yanamura) [inline dist/index.js:994]: I think its better to do this at the end of this code so that you dont have to increment manually

**Comment by yanamura** [comment]
@tatsuz0u  Would you apply this patch?
https://github.com/yanamura/ios-bump-version/commit/1f2161e3cb43c67bab5dfd9c1e7bd5bb122c9fa9

**Comment by tatsuz0u** [comment]
Yes, of course, I should have submitted a feature request instead because I'm not capable to deal with JavaScript codes.
I calculate and set the build number as outputs manually because I'm not sure if the process of bumping versions is async and I may get the number before bumping unexpectedly. And this approach is totally not elegant, not appropriate.

I sincerely apologize for bringing these not well-thought-out codes.
Please just feel free to do anything.

**Comment by yanamura** [comment]
Never mind, I am very grateful for your contribution!

Once I'll merge this PR and apply my patch, thanks!

**Comment by tatsuz0u** [comment]
It's very kind of you to say that, thanks. 😢

**Pull request merged**